Gopkg.lock

```diff
@@ -785,13 +952,39 @@
     "pkg/util/net/sets",
     "pkg/util/parsers",
     "pkg/util/pointer",
-    "pkg/util/taints"
+    "pkg/util/taints",
   ]
+  pruneopts = "UT"
   revision = "738a818620465158309cd7eb476543141f0fc7a2"
 
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "2ac72ff6af5667418b18c23f63d6317c04f251ae630ac6a85eeafe36d55e4e3e"
+  input-imports = [
+    "github.com/blang/semver",
+    "github.com/dlespiau/kube-test-harness",
+    "github.com/dlespiau/kube-test-harness/logger",
+    "github.com/getsentry/raven-go",
+    "github.com/jessevdk/go-flags",
+    "github.com/oklog/run",
+    "github.com/prometheus/client_golang/prometheus",
+    "github.com/prometheus/client_golang/prometheus/promhttp",
+    "github.com/sirupsen/logrus",
+    "github.com/spf13/pflag",
+    "github.com/stretchr/testify/assert",
+    "github.com/weaveworks/common/server",
+    "k8s.io/api/core/v1",
+    "k8s.io/apimachinery/pkg/apis/meta/v1",
+    "k8s.io/apimachinery/pkg/fields",
+    "k8s.io/apimachinery/pkg/util/yaml",
+    "k8s.io/apimachinery/pkg/watch",
+    "k8s.io/client-go/kubernetes",
+    "k8s.io/client-go/kubernetes/typed/core/v1",
+    "k8s.io/client-go/rest",
+    "k8s.io/client-go/tools/cache",
+    "k8s.io/client-go/tools/clientcmd",
+    "k8s.io/client-go/tools/clientcmd/api",
+    "k8s.io/kubernetes/pkg/controller/deployment/util",
+  ]
```

Comment:
The extent of the changes in this file is terrifying. Some look like they are possibly due to running a different version of `dep`. It would be good to separate these from the _actual_ changes.

Reply:
I wanted to upgrade github.com/spf13/pflag `1.0.0` -> `1.0.3` so ran:

`dep ensure -update github.com/spf13/pflag`

I'm unfamiliar w/ upgrading Gopkg.lock, one of the first hits on google was to this thread https://github.com/weaveworks/flux/pull/1334 :). I will review this a bit more before merging.